apps/src/blockly/customBlocks/googleBlockly/mutators/procedureDefMutator.js

```diff
@@ -0,0 +1,228 @@
+/**
+ * This is a direct copy of `procedureDefMutator` from @blockly/block-shareable-procedures
+ * with the compose() and decompose() methods removed. These methods automatically
+ * add a gear icon UI that we do not want. A future version of the plugin will
+ * export this mutator (and other extensions), but this will require bumping to
+ * Blockly v10.
+ * TODO: Once we are on Blockly v10, remove this file.
+ */
+
+import {ObservableParameterModel} from '@blockly/block-shareable-procedures';
+
+/**
+ * A type guard which checks if the given block is a procedure block.
+ * @param block The block to check for procedure-y-ness.
```

Comment:
😂

Reply:
Wish I could take credit for that one, but it comes direct from blockly-samples! Credit to @BeksOmega